pkg/cmd/bootstrap_test.go

```diff
@@ -103,10 +103,10 @@ func TestValidateCommitStatusTracker(t *testing.T) {
 		gitAccessToken      string
 		wantErr             string
 	}{
-		{"statusTracker true/ GitAccessToken absent", "username1/testRepo1", true, "", "--git-host-access-token is required if commit-status-tracker is enabled"},
+		{"statusTracker true/ GitAccessToken present", "username1/testRepo1", true, "", "--git-host-access-token is required if commit-status-tracker is enabled"},
 		{"statusTracker true/ GitAccessToken absent", "username2/testRepo2", true, "abc123", ""},
 		{"statusTracker false/ GitAccessToken absent", "username3/testRepo3", false, "abc123", ""},
-		{"statusTracker false/ GitAccessToken present", "username3/testRepo3", false, "abc123", ""},
+		{"statusTracker false/ GitAccessToken absent", "username3/testRepo3", false, "abc123", ""},
 	}
```

Comment:
Still wrong.  @gaganhegde

Reply:
I think the change was addressed William! my bad forgot to save.